bedrock/firefox/urls.py

```diff
@@ -85,6 +87,8 @@
     url(firstrun_re, views.FirstrunView.as_view(), name='firefox.firstrun'),
     url(whatsnew_re, views.WhatsNewRedirectorView.as_view(), name='firefox.whatsnew'),
     url(whatsnew_re_china, views.WhatsNewChinaView.as_view(), name='firefox.whatsnew.china'),
+    url(whatsnew_re_en, views.WhatsNewEnglishView.as_view(), name='firefox.whatsnew.en'),
```

Comment:
Nit: The other views are named after a country, but this one is named after a language. Is that intentional?

Reply:
Yeah, only because we're showing the same page in US, UK, and Canada. Naming it for a country seemed wrong when there are three countries getting it.